main.tf

```
@@ -1,20 +1,56 @@
-resource "scaleway_object_bucket" "this" {
+resource "scaleway_object_bucket" "main" {
```

Ok to renamed, not adding a "moved" block will break users. Should be:
```suggestion
moved {
  from = scaleway_object_bucket.this
  to   = scaleway_object_bucket.main
}

resource "scaleway_object_bucket" "main" {
```